README.md

```diff
@@ -360,6 +360,9 @@ buildctl build ... \
 * `mode=max`: export all the layers of all intermediate steps.
 * `ref=docker.io/user/image:tag`: reference
 * `oci-mediatypes=true|false`: whether to use OCI mediatypes in exported manifests. Since BuildKit `v0.8` defaults to true.
+* `compression=[uncompressed,gzip,estargz,zstd]`: choose compression type for layers newly created and cached, gzip is default value. estargz and zstd should be used with `oci-mediatypes=true`.
+* `compression-level=[value]`: compression level for gzip, estargz (0-9) and zstd (0-22)
+* `force-compression=true`: forcibly apply `compression` option to all layers.
```

Comment:
compression-force

Reply:
Thank you for the review.
This  follows the naming convention of `--output` option which have `force-compression`.
Let's rename them to `compression-force` (and deprecate `force-compression`) in a separated PR.